src/networkStats.ts

```diff
@@ -0,0 +1,36 @@
+const IMAGE_URL = "https://portal.excalidraw.com/test128.png";
+const IMAGE_SIZE = 35747; // in bytes
+const calculateSpeed = (startTime: number, endTime: number) => {
+  const duration = (endTime - startTime) / 1000;
+  const imageSizeInBits = IMAGE_SIZE * 8;
+  let speed = imageSizeInBits / duration;
+  const suffix = ["bps", "kbps", "mbps", "gbps"];
+  let index = 0;
+  while (speed > 1024) {
+    index++;
+    speed = speed / 1024;
```

Comment:
Let's do the 1000 ones:

- `bit/s`
- `kbit/s`
- `Mbit/s`
- `Gbit/s`


source: https://en.wikipedia.org/wiki/Data-rate_units#Conversion_table